### DISCUSSION TITLE
Is it possible to run the monai bundle model inference on CPU

### DISCUSSION THREAD

**Discussion by Bala93** [Project-MONAI/model-zoo#305] [Q&A]
I am trying to run the inference of https://github.com/Project-MONAI/model-zoo/tree/dev/models/brats_mri_segmentation. I have only 12GB GPU, and it is resulting in out-of-memory error. Is it possible to run the inference in CPU ?

**Comment by Bala93** [comment]
The roi size in json config file could be modified to fit the data in the gpu.

**Reply by yiheng-wang-nv** [reply]
Hi @Bala93 , you can modify this line:
https://github.com/Project-MONAI/model-zoo/blob/dev/models/brats_mri_segmentation/configs/inference.json#L12
to cpu. In addition, as you said, you can also reduce the roi size.
